tensorflow/g3doc/how_tos/tool_developers/index.md

```diff
@@ -28,17 +28,15 @@ by calling `as_graph_def()`, which returns a `GraphDef` object.
 
 The GraphDef class is an object created by the ProtoBuf library from the
 definition in
-[tensorflow/core/framework/graph.proto](https://github.com/tensorflow/tensorflow
-/blob/master/tensorflow/core/framework/graph.proto). The protobuf tools parse
+[tensorflow/core/framework/graph.proto](https://github.com/tensorflow/tensorflow/blob/master/tensorflow/core/framework/graph.proto). The protobuf tools parse
```

Comment:
Clearly this is not this PRs fault, but the link should point to tensorflow.org/code/tensorflow/core/framework/graph.proto